src/test/java/io/jenkins/plugins/opentelemetry/rules/CheckIsDockerAvailable.java

```diff
@@ -1,19 +0,0 @@
-/*
- * Copyright The Original Author or Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-package io.jenkins.plugins.opentelemetry.rules;
-
-import org.junit.Assume;
-import org.junit.rules.ExternalResource;
-import org.testcontainers.DockerClientFactory;
-
-/**
- * Rule to check if Docker is available.
- */
-public class CheckIsDockerAvailable extends ExternalResource {
```

Comment:
I guess why, but it needs a why in the PR comments

Reply:
Unused / dead code.